data/json/items/resources/metal.json

```diff
@@ -12,7 +12,7 @@
     "material": [ "steel" ],
     "qualities": [ [ "HAMMER", 1 ] ],
     "techniques": [ "WBLOCK_1" ],
-    "volume": "1 L",
+    "volume": "370 ml",
```

Comment:
I think you missed adding longest side to the regular pipe

Reply:
I added it, but I did some undo and it vanished. Thanks.